PhpcrMigration/Infrastructure/Repository/EntityRepository.php

```diff
@@ -193,4 +209,45 @@ public function addOrUpdateChildNode(array $data, string $tableName, array $type
 
         $this->connection->insert($tableName, $data, $types);
     }
+
+    private function getNextIdValue(string $tableName): ?int
+    {
+        $result = null;
+
+        $sequences = $this->getSequences();
+        foreach ($sequences as $sequence) {
+            $sequenceName = $sequence->getName();
+            if (\str_contains($sequenceName, $tableName)) {
+                /** @var int|null|false $result */
+                $result = $this->connection->fetchOne("SELECT nextval('$sequenceName')");
```

Comment:
should we go with:

```php
return $this->connection->getPlatform()->getSequenceNextValSQL($sequenceName);
```

This would work for eventually other platforms.

Reply:
updated it